web/themes/custom/nysenate_theme/src/patterns/components/hero/hero.twig

```diff
@@ -19,7 +19,7 @@
         <div>
           <h2 class="c-senator-hero--title">
             {% if not is_active %}former
-            {% endif %}new york state senator<br>
+            {% endif %}New York State Senator<br>
```

Comment:
Should "former" also receive capitalization?

Reply:
I agree that "Former" should be capitalized, but I'm not going to hold up the deployment for such a change.